OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -283,21 +285,23 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		abstract class MovePart : Activity
+		// NOTE: This class is uncancellable until it completes its intended move...
+		// If we could solve this, we could enable path smoothing!
+		public abstract class MovePart : Activity
```

Comment:
Never been a fan of internal private classes. That makes them just harder to understand from the outside. Besides, I have to say I did that because I planned to get rid of them sooner or later once I could separate responsibilities with tests. I would not make them private again, but if they hurt your eyes then I'll make them back to private.